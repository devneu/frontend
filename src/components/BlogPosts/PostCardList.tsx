import React from 'react';
import { Col, Row } from 'antd';
import SmallPostCard from './SmallPostCard';

interface PostCardListProps {
  postInfo: {
    id: number;
    postImg: string;
    postTitle: string;
    postText: string;
    postDate: string;
    sizeBlock: number;
  };
}

const PostCardList = ({ postInfo }: PostCardListProps) => (
  <Row justify="space-around" align="middle">
    <Col span={14}>
      <Row justify="space-between">
        {
          // @ts-ignore
          // eslint-disable-next-line react/destructuring-assignment,max-len
          postInfo.map(
            (
              obj: {
                id?: number;
                postImg?: string;
                postDate?: string;
                postTitle?: string;
                postText?: string;
              },
              index: number,
            ) => (
              <Col span={index === 0 ? 24 : 11}>
                <SmallPostCard key={obj.id} post={obj} />
              </Col>
            ),
          )
        }
      </Row>
    </Col>
  </Row>
);

export default PostCardList;
