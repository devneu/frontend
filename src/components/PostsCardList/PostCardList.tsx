import React from 'react';
import { Col, Row } from 'antd';
import SmallPostCard from '../Posts/SmallPostCard';
import { Post } from '../../types/Post.interface';
import FeaturePost from '../Posts/FeaturePost';

interface PostsCardListProps {
  posts: Post[];
  featurePost: Post;
}

const PostsCardList = ({ posts, featurePost }: PostsCardListProps) => (
  <Row gutter={[24, 24]}>
    <Col span={24}>
      <FeaturePost post={featurePost} />
    </Col>
    {posts.map((post) => (
      <Col span={12} key={post.id}>
        <SmallPostCard post={post} />
      </Col>
    ))}
  </Row>
);

export default PostsCardList;
