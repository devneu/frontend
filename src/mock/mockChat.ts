export const photos = [
  {
    img: 'https://images.unsplash.com/photo-1469854523086-cc02fe5d8800?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8M3x8fGVufDB8fHx8&w=1000&q=80',
  },
  {
    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSV_EfZmXNnuKrtvDecfz6B7r955J6X7EFTsw&usqp=CAU',
  },
  {
    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrOELmQBbTGeOYYBHmT6UxFVma2GOPE-6kpg&usqp=CAU',
  },
  {
    img: 'https://sourceduk.com/wp-content/uploads/2018/10/girlgoneabroad.jpg',
  },
  {
    img: 'https://www.workandtravelgroup.com/wp-content/uploads/wtg-group-2.jpg',
  },
  {
    img: 'https://www.moneycrashers.com/wp-content/uploads/2018/10/travel-world-map-1068x713.jpg',
  },
];

export const attachments = [
  {
    name: 'pdf',
    img: 'https://www.iconpacks.net/icons/2/free-pdf-icon-3385.png',
    description: 'how to create a case study',
  },
  {
    name: 'zip',
    img: 'https://www.iconpacks.net/icons/2/free-zip-icon-1519-thumb.png',
    description: 'Project Documents',
  },
  {
    name: 'word',
    img: 'https://www.pngkey.com/png/full/210-2104640_imagen-microsoft-office-365-word-logo.png ',
    description: 'Competitor Analysis Template',
  },
];

export const avatar = {
  img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
};

export const groupData = {
  img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
  name: 'Nasir Uddin',
  description: 'Ux Designer',
  icon: 'https://i.pinimg.com/originals/44/9d/d1/449dd1f38a8bd87b41f7b960db0fefff.png',
  memberImgs: [
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://www.w3schools.com/howto/img_avatar2.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://www.w3schools.com/howto/img_avatar.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://www.w3schools.com/w3images/avatar6.png',
    },
  ],
};
