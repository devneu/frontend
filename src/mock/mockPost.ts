export const createMockPost = (options = {} as any) => ({
  id: options.id || 999,
  img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
  title: 'DEV-17.com',
  date: 'September 23, 2010',
  text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
  articleText: `
    # I have odd cosmic thoughts every day
    
    For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.
    
    Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
  tag: ['web', 'js'],
});

export const createMockPosts = () => [
  {
    id: 1,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
  {
    id: 2,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
  {
    id: 3,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
  {
    id: 4,
    img: 'https://dummyimage.com/700x350/dee2e6/6c757d.jpg',
    title: 'PostPage Title',
    text: `Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.`,
    date: 'September 23, 2010',
    articleText: `
# I have odd cosmic thoughts every day

For me, the most fascinating interface is Twitter. I have odd cosmic thoughts every day and I realized I could hold them to myself or share them with people who might be interested.

Venus has a runaway greenhouse effect. I kind of want to know what happened there because we're twirling knobs here on Earth without knowing the consequences of it. Mars once had running water. It's bone dry today. Something bad happened there as well.
    `,
    tag: ['Web Design', 'Freebies'],
  },
];
