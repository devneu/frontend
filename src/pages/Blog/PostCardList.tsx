import {Col, Row} from "antd";
import {postInfo} from "./data";
import SmallPostCard from "./SmallPostCard";
import React from "react";

const PostCardList  = () => {
    return (
        <Row justify="space-around" align="middle">
            <Col span={14}>
                <Row justify="space-between">
                    {
                        postInfo.map((obj, index) => <Col span={index==0 ? 24: 11}><SmallPostCard key={obj.id} post={obj}/></Col>)
                    }
                </Row>
            </Col>
            <Col span={6}></Col>
        </Row>
    );
}

export default PostCardList;