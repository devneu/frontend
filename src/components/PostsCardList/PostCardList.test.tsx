import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import PostCardList from './PostCardList';
import { createMockPost, createMockPosts } from '../../mock/mockPost';

describe('PostCardList', () => {
  it('Should successfully render', () => {
    const wrapper = renderer
      .create(
        <BrowserRouter>
          <PostCardList posts={createMockPosts()} featurePost={createMockPost()} />
        </BrowserRouter>,
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
