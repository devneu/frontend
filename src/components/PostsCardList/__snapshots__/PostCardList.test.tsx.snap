// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostCardList Should successfully render 1`] = `
<div
  className="ant-row"
  style={
    Object {
      "marginBottom": -12,
      "marginLeft": -12,
      "marginRight": -12,
      "marginTop": -12,
    }
  }
>
  <div
    className="ant-col ant-col-24"
    style={
      Object {
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      }
    }
  >
    <div
      className="ant-card ant-card-bordered post-feature-card"
    >
      <div
        className="ant-card-cover"
      >
        <img
          alt="card-img"
          src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
        />
      </div>
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <div
          className="ant-card-meta"
        >
          <div
            className="ant-card-meta-detail"
          >
            <div
              className="ant-card-meta-description"
            >
              September 23, 2010
            </div>
          </div>
        </div>
        <h2
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          DEV-17.com
        </h2>
        <div
          className="ant-typography post-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
        </div>
        <a
          aria-current={null}
          href="/blog/999"
          onClick={[Function]}
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            type="button"
          >
            <span>
              Read more
            </span>
            <span
              aria-label="arrow-right"
              className="anticon anticon-arrow-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="arrow-right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-12"
    style={
      Object {
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      }
    }
  >
    <div
      className="ant-card ant-card-bordered post-card"
    >
      <div
        className="ant-card-cover"
      >
        <img
          alt="card-img"
          src="https://dummyimage.com/850x350/dee2e6/6c757d.jpg"
        />
      </div>
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <div
          className="ant-card-meta"
        >
          <div
            className="ant-card-meta-detail"
          >
            <div
              className="ant-card-meta-description"
            >
              September 23, 2010
            </div>
          </div>
        </div>
        <h2
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Featured PostPage Title
        </h2>
        <div
          className="ant-typography post-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
        </div>
        <a
          aria-current={null}
          href="/blog/0"
          onClick={[Function]}
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            type="button"
          >
            <span>
              Read more
            </span>
            <span
              aria-label="arrow-right"
              className="anticon anticon-arrow-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="arrow-right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-12"
    style={
      Object {
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      }
    }
  >
    <div
      className="ant-card ant-card-bordered post-card"
    >
      <div
        className="ant-card-cover"
      >
        <img
          alt="card-img"
          src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
        />
      </div>
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <div
          className="ant-card-meta"
        >
          <div
            className="ant-card-meta-detail"
          >
            <div
              className="ant-card-meta-description"
            >
              September 23, 2010
            </div>
          </div>
        </div>
        <h2
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          PostPage Title
        </h2>
        <div
          className="ant-typography post-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
        </div>
        <a
          aria-current={null}
          href="/blog/1"
          onClick={[Function]}
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            type="button"
          >
            <span>
              Read more
            </span>
            <span
              aria-label="arrow-right"
              className="anticon anticon-arrow-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="arrow-right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-12"
    style={
      Object {
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      }
    }
  >
    <div
      className="ant-card ant-card-bordered post-card"
    >
      <div
        className="ant-card-cover"
      >
        <img
          alt="card-img"
          src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
        />
      </div>
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <div
          className="ant-card-meta"
        >
          <div
            className="ant-card-meta-detail"
          >
            <div
              className="ant-card-meta-description"
            >
              September 23, 2010
            </div>
          </div>
        </div>
        <h2
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          PostPage Title
        </h2>
        <div
          className="ant-typography post-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
        </div>
        <a
          aria-current={null}
          href="/blog/2"
          onClick={[Function]}
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            type="button"
          >
            <span>
              Read more
            </span>
            <span
              aria-label="arrow-right"
              className="anticon anticon-arrow-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="arrow-right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-12"
    style={
      Object {
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      }
    }
  >
    <div
      className="ant-card ant-card-bordered post-card"
    >
      <div
        className="ant-card-cover"
      >
        <img
          alt="card-img"
          src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
        />
      </div>
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <div
          className="ant-card-meta"
        >
          <div
            className="ant-card-meta-detail"
          >
            <div
              className="ant-card-meta-description"
            >
              September 23, 2010
            </div>
          </div>
        </div>
        <h2
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          PostPage Title
        </h2>
        <div
          className="ant-typography post-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
        </div>
        <a
          aria-current={null}
          href="/blog/3"
          onClick={[Function]}
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            type="button"
          >
            <span>
              Read more
            </span>
            <span
              aria-label="arrow-right"
              className="anticon anticon-arrow-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="arrow-right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-12"
    style={
      Object {
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      }
    }
  >
    <div
      className="ant-card ant-card-bordered post-card"
    >
      <div
        className="ant-card-cover"
      >
        <img
          alt="card-img"
          src="https://dummyimage.com/700x350/dee2e6/6c757d.jpg"
        />
      </div>
      <div
        className="ant-card-body"
        style={Object {}}
      >
        <div
          className="ant-card-meta"
        >
          <div
            className="ant-card-meta-detail"
          >
            <div
              className="ant-card-meta-description"
            >
              September 23, 2010
            </div>
          </div>
        </div>
        <h2
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          PostPage Title
        </h2>
        <div
          className="ant-typography post-text"
          style={
            Object {
              "WebkitLineClamp": undefined,
            }
          }
        >
          Science is an enterprise that should be cherished as an activity of the free human mind. 
    Because it transforms who we are, how we live, and it gives us an understanding of our place in the universe. 
    The universe is large and old, and the ingredients for life as we know it are everywhere, so there's no reason to think that Earth would be unique in that regard. Whether of not the life became intelligent is a different question, and we'll see if we find that. 
    If you get asteroids about a kilometer in size, those are large enough and carry enough energy into our system to disrupt transportation, communication, the food chains, and that can be a really bad day on Earth.
        </div>
        <a
          aria-current={null}
          href="/blog/4"
          onClick={[Function]}
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            type="button"
          >
            <span>
              Read more
            </span>
            <span
              aria-label="arrow-right"
              className="anticon anticon-arrow-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="arrow-right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                />
              </svg>
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
</div>
`;
